app/lib/hooks/useFeatureFlags.ts

```diff
@@ -0,0 +1,29 @@
+import { IS_PREVIEW_BUILD, IS_PRODUCTION } from "lib/constants";
+import { useMemo } from "react";
+
+type FeatureFlags = {
```

Comment:
probably a good idea to maintain a strict type like this 👍 as these flags will be imported all over the app, ts can warn us when we forget to clean something up.

Reply:
💯!! Left a note to remind us to clean up feature flags too when features are live and stable for a while.